docs/guidelines.md

```diff
@@ -25,6 +23,14 @@ example, **MUST** be without comment means that the website could crash if an
 analyzer sends a comment anyway. **SHOULD** indicates any consumer of the output
 must still guard against unwanted behaviour.
 
+### Approvability
+
+A solution is approvable if it either matches the (set of) optimal solution(s)
+of that exercise, or if it shows understanding of the teaching goals of that
+exercise and every _core_ exercise that came before. This means that an
+analyzer **MAY** approve a solution but provide a set of comments as TIPS or
```

Comment:
Could we either say "tips relating to the exercise," or get rid of "tips or" pls? I don't want to encourage/support non-exercise related tips through the analyzers.

Reply:
Yes!